.github/workflows/tiers.yaml

```diff
@@ -0,0 +1,61 @@
+name: Run tier tests
+
+on:
+  workflow_call:
+    inputs:
+      tier:
+        type: string
+      pr_number:
+        required: true
+        type: string
+      repo:
+        required: true
+        type: string
+      head_ref:
+        required: true
+        type: string
+    outputs:
+      tier1-result:
+        description: "Whether Tier1 tests passed"
+        value: ${{ jobs.tier1.outputs.result }}
+      tier2-result:
+        description: "Whether Tier1 tests passed"
+        value: ${{ jobs.tier2.outputs.result }}
+    secrets:
+      token:
+        description: "Github access token"
+        required: true
+      testing_farm_token:
+        description: "Testing farm access token"
+        required: true
+
+jobs:
+  tier1:
+    name: Run Tier-1 tests
+    uses: ./.github/workflows/run-on-testing-farm.yaml
+    with:
+      distro: "fedora-37"
+      plan: "tier1"
+      pr_number: ${{ inputs.pr_number }}
+      repo: ${{ inputs.repo }}
+      head_ref: ${{ inputs.head_ref }}
+    secrets:
+      token: ${{ secrets.token }}
+      testing_farm_token: ${{ secrets.testing_farm_token }}
+  tier2:
+    name: Run Tier-2 tests
+    needs: tier1
+    if: contains(${{ inputs.tier }}, "tier2")
+    strategy:
+      matrix:
+        distro: ["fedora-37", "fedora-38"]
```

Comment:
and here.. See https://api.dev.testing-farm.io/v0.1/composes/public for available composes.

Reply:
Done